Observer/AddToCart.php

```diff
@@ -70,7 +70,7 @@ private function addToCart($product, $quantity)
         if ($product->getId() != $childProduct->getId()) {
             $product = $this->productRepository->getById($product->getId());
             // for legacy reasons - we have been passing the SKU as ID for the child products
-            $data['option_id'] = $childProduct->getSku();
+            $data['option_id'] = $childProduct->getSku() ? $childProduct->getSku() : $childProduct->getId();
```

Comment:
Lets make it consistent with the previous [pull request](https://github.com/Metrilo/metrilo-magento2-integration/pull/73/files#diff-3239ea2def5b5dfe803295a77f337128R18-R20).
```suggestion
            $sku = $childProduct->getSku();
            $data['option_id'] = $sku() ? $sku() : $childProduct->getId();
```

Reply:
Refactored.